src/rov/pca9685/src/main_interactive.cpp

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <map>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/lexical_cast.hpp>
+
+#include "pca9685/ServoDriver.hpp"
+
+int main(int argc, char** argv) {
```

Comment:
Looks like a handy tester program. It really *feels* like there should have been a more ROSy way, but everything I'm coming up with involves massive command line (ew) or some kinda over-the-top RVIZ integration.
Ah, that's what I probably would've done differently: I probably would've made a separate node written in Python because Ooga Booga I like Python and I'mma take every chance I can get to use it.
But yeah, looks good.

Reply:
This is the only way I could find of easily splitting the functionality into a library and two executables. Else I would have had to create another ROS package.....ew.